ursgal/ursgal
# Change CI/CD settings

- start tests with tox
   - test python 3.5 to 3.7
   - test building docs
   - run example_scripts
   - optionally check coding style

 - add setup.cfg to build docs using setup.py
 - add development requirements
 - setup.py reads requirements and installs them with setup.py install/develop
 - make  development compatible with tox
 - make travis autodeploy when new tag is set
